cowamm/dashboard/volume_evolution_4346160.sql

```diff
@@ -0,0 +1,25 @@
+--Query computes the trading volume over all Cow AMMs (matching users and rebalancing)
+
+-- Parameters:
+-- {{frequence}}: the frequence of the data, e.g. 'day', 'week', 'month'
+with all_trades as (
+    select * from cow_protocol_ethereum.trades
+    union all
+    select * from cow_protocol_gnosis.trades
+    union all
+    select * from cow_protocol_arbitrum.trades
+    union all
+    select * from cow_protocol_base.trades
+),
+
+cow_trades as (
+    select
+        date_trunc('{{frequence}}', block_date) as period,
```

Comment:
If we add this here, we should probably also add it for the TVL evolution to not cause confusion like [this](https://cowservices.slack.com/archives/C067ZLV8U13/p1736488936764779?thread_ts=1736156982.669889&cid=C067ZLV8U13)